dts/arm/atmel/samr34.dtsi

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021 Argentum Systems Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <atmel/saml21.dtsi>
+#include <dt-bindings/lora/sx126x.h>
+
+#include "saml21.dtsi"
+
+/ {
+	aliases {
+		lora0 = &lora;
+	};
+
+	soc {
+		pinctrl@40002800 {
+			ranges = <0x40002800 0x40002800 0x180>;
+
+			portc: gpio@40002900 {
+				compatible = "atmel,sam0-gpio";
+				reg = <0x40002900 0x80>;
+				label = "PORTC";
+				gpio-controller;
+				#gpio-cells = <2>;
+				#atmel,pin-cells = <2>;
+			};
+		};
+	};
+};
+
+/delete-node/ &dac;
+
+&sercom4 {
+	/* SERCOM4 is used to interface with the internal LoRa radio */
+	compatible = "atmel,sam0-spi";
+	dipo = <0>;
+	dopo = <1>;
+	cs-gpios = <&portb 31 GPIO_ACTIVE_LOW>;
```

Comment:
Is this really correct? This doesn't look like it belongs in a SoC .dtsi file; by definition GPIOs are exterior to the SoC itself.

I would have the same question for any other similar types of settings in this PR.

Reply:
The SAMR34 / SAMR35 is actually a SiP (System-in-Package) - these pins are internally connected between the SAML21 silicon and the SX127x silicon, and have no other functional uses... so I believe in this case, this is indeed correct.